lib/osc-loader.js

```diff
@@ -30,6 +30,18 @@ export default class OscLoader extends EventEmitter {
                     this.editors.goTo(resultDict['row'] - 1, resultDict['column'])
                 }
 
+                if (resultDict['tab'] !== undefined) {
+                    atom.workspace.getPanes()[0].setActiveItem(atom.workspace.getTextEditors()[resultDict['tab']])
+                }
+
+                if (resultDict['search']) {
+                    const lineNumber = this.editors.currentEditor().getText().split("\n").findIndex(line => line.includes(resultDict['search']));
+                    if (lineNumber > -1)  {
+                      this.editors.goTo(lineNumber, 0)
+                      this.editors.currentEditor().setScrollTopRow(lineNumber)
+                    }
+                }
+
```

Comment:
I know that osc-loader has never been tested, but it would be possible to add a test for this feature?

Reply:
I will have a look. Shouldn't be a big deal